core/lib/spree/testing_support/dummy_app.rb

```diff
@@ -64,7 +65,11 @@ class Application < ::Rails::Application
 
     if RAILS_52_OR_ABOVE
       config.action_controller.default_protect_from_forgery = true
-      config.active_record.sqlite3.represent_boolean_as_integer = true
+      unless RAILS_6_OR_ABOVE
+        # `.represent_boolean_as_integer=` is always true.
+        # Setting this is deprecated and will be removed in Rails 6.1
```

Comment:
I think moving this comment into the commit message description is enough documentation.